src/test/java/de/caritas/cob/userservice/api/manager/consultingtype/ConsultingTypeManagerTest.java

```diff
@@ -72,8 +73,17 @@ void isConsultantBoundedToAgency_Should_Return_False()
   @Test
   void getAllConsultingTypeIds_Should_Return_The_Same_List()
```

Comment:
please also add a test for non-mulititenant context to make sure, this will always work, even if we disable feature toggle